bool emerg_stop
bool emerg_speed_up
